aci/resource_aci_fvrsbulkpathatt.go

```diff
@@ -0,0 +1,329 @@
+								"regular",
+								"native",
+								"untagged",
+							}, false),
+						},
+						"primary_encap": &schema.Schema{
+							Type:     schema.TypeString,
+							Optional: true,
+							Computed: true,
+						},
+					},
+				},
+			},
+		},
+	}
+}
+
+func resourceAciBulkStaticPathCreate(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
+	log.Printf("[DEBUG] BulkStaticPath: Beginning Creation")
+
+	aciClient := m.(*client.Client)
+
+	ApplicationEPGDn := d.Get("application_epg_dn").(string)
+	staticPathSet := staticPathPayload(d.Get("static_path").(*schema.Set).List(), "create")
+
+	contentMap := make(map[string]interface{})
+	contentMap["dn"] = ApplicationEPGDn
+	cont, err := preparePayload(className, toStrMap(contentMap), staticPathSet)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	_, diags := bulkStaticPathRequest(aciClient, "POST", ApplicationEPGDn, cont)
+	if diags.HasError() {
+		return diags
+	}
+	d.SetId(ApplicationEPGDn)
+	log.Printf("[DEBUG] %s: Creation finished successfully", d.Id())
+
+	return resourceAciBulkStaticPathRead(ctx, d, m)
+}
+
+func resourceAciBulkStaticPathRead(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
+	log.Printf("[DEBUG] %s: Beginning Read", d.Id())
+
+	aciClient := m.(*client.Client)
+
+	dn := d.Id()
+	bulkStaticPath, diags := getAciBulkStaticPath(aciClient, dn)
+	if diags.HasError() {
+		return diags
+	}
+	d = setAciBulkStaticPath(bulkStaticPath, d)
+	d.Set("application_epg_dn", dn)
+
+	log.Printf("[DEBUG] %s: Read finished successfully", d.Id())
+
+	return nil
+}
+
+func resourceAciBulkStaticPathUpdate(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
+	log.Printf("[DEBUG] %s: Beginning Update", d.Id())
+
+	aciClient := m.(*client.Client)
+
+	ApplicationEPGDn := d.Get("application_epg_dn").(string)
+
+	old_static_path, new_static_path := d.GetChange("static_path")
+	del := diffInStaticPath(old_static_path.(*schema.Set), new_static_path.(*schema.Set))
+
+	createPayload := staticPathPayload(new_static_path.(*schema.Set).List(), "create")
+	deletePayload := staticPathPayload(del, "delete")
+	var payload []interface{}
+	for _, createObj := range createPayload {
+		payload = append(payload, createObj)
+	}
+	for _, deleteObj := range deletePayload {
+		payload = append(payload, deleteObj)
+	}
+
+	contentMap := make(map[string]interface{})
+	contentMap["dn"] = ApplicationEPGDn
+
+	cont, err := preparePayload(className, toStrMap(contentMap), payload)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	_, diags := bulkStaticPathRequest(aciClient, "POST", ApplicationEPGDn, cont)
+	if diags.HasError() {
+		return diags
+	}
+	d.SetId(ApplicationEPGDn)
+	log.Printf("[DEBUG] %s: Update finished successfully", d.Id())
+	return resourceAciBulkStaticPathRead(ctx, d, m)
+}
+
+func getAciBulkStaticPath(aciClient *client.Client, dn string) ([]interface{}, diag.Diagnostics) {
```

Comment:
Is this returning 1 or more static paths? If more than one, rename the function to getAciBulkStaticPaths